updating plotBEMIO for multiple hydro solutions

kmruehl opened the pull request:
This PR adds the [Compare_BEM](https://github.com/WEC-Sim/WEC-Sim/projects/58#card-77446108) functionality. @akeeste this should resolve the nested varargin bug.
- [x] resolve merge conflicts with bemioCleanup branch
- [x] update Compare_BEM naming conventions to adhere to branch
- [x] verify that Compare_BEM works for all WECs (not just RM3)
- [x] extend to more than two BEM cases

jtgrasb committed:
Fix AQWA data for sphere, ellipsoid, and WEC3

kmruehl committed:
working on compare BEM hydro data

kmruehl committed:
combined added mass BEM

kmruehl committed:
compare added mass complete

kmruehl committed:
removing duplicate files

kmruehl committed:
updating plot radiation damping

kmruehl committed:
updating plot radiation irf

kmruehl committed:
updating plot exc mag

kmruehl committed:
updated plot excitation phase and irf

kmruehl committed:
updated plot bemio scripts

kmruehl committed:
reverting bemio examples

kmruehl committed:
updating plot functions to camelCase

kmruehl committed:
resolving conflicts

kmruehl committed:
updating compare plots for multiple body wecs

kmruehl committed:
extending plotAddedMass to multiple hydro datasets

kmruehl committed:
extending plotRadDamping to multiple hydro datasets

kmruehl committed:
adding extensible plot rad irf

kmruehl committed:
adding exc mag and phase

kmruehl committed:
updaing exc irf plot for mult bemio results

kmruehl committed:
resolve radiationIRF SS bug

akeeste committed:
Merge branch 'bemioCleanup' into compareBEMIO

kmruehl committed:
cleaning up compareBEM

kmruehl committed:
resolving nested varargin for plotAddedMass

kmruehl committed:
resolving nested varargin for all plots

kmruehl committed:
nested varargin bug resolved

kmruehl committed:
updating compareBEM to camelCase

kmruehl committed:
resolving plotExcIRF bug

kmruehl committed:
removing pngs

akeeste reviewed (commented):
@kmruehl Thanks for this fix! A few questions:

- What was the to the AQWA hydrodata? Is this going to conflict with the dev branch now?
- what is readBEMIOH5 doing? I changed a few of these functions in my PR and am getting mixed up now

I don't think Github is comparing these files correctly. It is showing a lot of changes that should have already been incorporated in your last PR

kmruehl commented:
> @kmruehl Thanks for this fix! A few questions:
> 
>     * What was the to the AQWA hydrodata? Is this going to conflict with the dev branch now?
This is from Jeff's PR and need to be removed prior to dev merge, but needed now for testing
> 
>     * what is readBEMIOH5 doing? I changed a few of these functions in my PR and am getting mixed up now
Moved this function from bodyClass to source/functions/BEMIO, where we want it long-term. It should be called from the bodyClass prior to merging with dev
> 
> 
> I don't think Github is comparing these files correctly. It is showing a lot of changes that should have already been incorporated in your last PR
